### PR TITLE
fix(AdvancedDataCard): fix hover on actions removing hover on card

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1496)

### CODE REVIEW

File: src/community/advanced-data-card.tsx [hunk 1]
```diff
@@ -282,65 +284,69 @@ export const AdvancedDataCard = React.forwardRef<HTMLDivElement, AdvancedDataCar
...
-                                styles.cardContentStyle,
-                                !hasFooter && !hasExtras ? styles.minHeight : ''
-                            )}
+                <Boxed className={styles.dataCard}>
+                    <div className={styles.touchableContainer}>
+                        <Touchable
+                            onPress={onPress}
+                            tabIndex={0}
+                            maybe
+                            className={classNames(styles.touchable)}
```
Comment (atabel) [inline src/community/advanced-data-card.tsx:293]:
```suggestion
                            className={styles.touchable}
```

File: src/community/advanced-data-card.tsx [hunk 2]
```diff
@@ -282,65 +284,69 @@ export const AdvancedDataCard = React.forwardRef<HTMLDivElement, AdvancedDataCar
...
+                                                        <Divider />
+                                                    </Box>
+                                                )}
+                                            </div>
+                                        );
+                                    })}
+                                </Box>
+                            )}
+                        </Touchable>
+                        <CardActionsGroup actions={actions} onClose={onClose} padding={7} />
```
Comment (atabel) [inline src/community/advanced-data-card.tsx:346]: why 7? weird number

Comment (marcoskolodny) [reply src/community/advanced-data-card.tsx:351]: Added a comment to explain why we used 7px in here.

**Review by aweell — APPROVED** [review]
LGTM!

**Pull request merged**